sale_order_line_variant_description/views/product_view.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- © 2015 Alex Comba - Agile Business Group
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html). -->
+     
+<openerp>
+    <data>
```

Comment:
Please remove data tag.
See https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-10.0 as reference.